orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/load_table.py

```diff
@@ -211,178 +227,80 @@ def start_load(
     )
 
     if num_new_rows > 0:
-        context.log.info(f"{num_new_rows} for {metadata_type}")
-        job_id = _ingest_table(
-            data_repo_client,
+        _ingest_table(
+            data_repo_service,
             target_hca_dataset,
             metadata_type,
             scratch_config
         )
-        yield Output(job_id, "job_id")
 
-    yield Output("no_job", "no_job")
+    return num_new_rows
 
 
 def _get_outdated_ids(
         table_name: str,
-        target_hca_dataset: HcaDataset,
+        target_hca_dataset: TdrDataset,
         scratch_config: ScratchConfig,
-        scratch_dataset_name: HcaScratchDatasetName,
-        outdated_ids_table_name: str,
         bigquery_service: BigQueryService
-) -> None:
+) -> str:
     fq_dataset_id = target_hca_dataset.fully_qualified_jade_dataset_name()
     jade_table = f"{target_hca_dataset.project_id}.{fq_dataset_id}.{table_name}"
-    destination = f"{scratch_dataset_name}.{outdated_ids_table_name}"
+    out_path = f"{scratch_config.scratch_area()}/outdated-ids/{table_name}"
 
     query = f"""
+    EXPORT DATA OPTIONS(
+        uri='{out_path}/*',
+        format='CSV',
+        overwrite=true
+    ) AS
     WITH latest_versions AS (
         SELECT {table_name}_id, MAX(version) AS latest_version
-        FROM {jade_table} GROUP BY {table_name}_id
+        FROM `{jade_table}` GROUP BY {table_name}_id
     )
     SELECT J.datarepo_row_id FROM
-        {jade_table} J JOIN latest_versions L
+        `{jade_table}` J JOIN latest_versions L
         ON J.{table_name}_id = L.{table_name}_id
     WHERE J.version < L.latest_version
     """
 
-    bigquery_service.build_query_job_with_destination(
+    bigquery_service.run_query(
         query,
-        destination,
-        scratch_config.scratch_bq_project
-    ).result()
-
-
-def _export_outdated(
-        metadata_type: MetadataType,
-        outdated_ids_table_name: str,
-        scratch_dataset_name: HcaScratchDatasetName,
-        scratch_config: ScratchConfig,
-        bigquery_service: BigQueryService
-) -> None:
-    out_path = f"{scratch_config.scratch_area()}/outdated-ids/{metadata_type}/*"
```

Comment:
Removing this code is what the `EXPORT DATA OPTIONS` shortcut mentioned above gets us.